setup.py

```diff
@@ -36,9 +36,9 @@
 #   - https://github.com/python/mypy
 class build_py(_build_py):
     def pin_version(self):
-        path = os.path.join(self.build_lib, "dvc")
-        self.mkpath(path)
```

Comment:
A little worried about using our mkdir instead of self.mkpath, as it might have some hidden logic/mode/etc in it.

Reply:
fixed